### ISSUE TITLE
Installation issues on iOS and Android

### ISSUE THREAD

**Issue by esamelson** [invertase/react-native-firebase#266]
First of all, thanks to everyone for all of your great work on this library!

While installing, I ran into a couple of issues that weren't documented and took me a while to track down. They could be related to the fact that I'm on a slightly older version of RN. I am wondering if you would accept a PR adding any or all of these issues to the docs.

1) Android - I was getting this error when including `compile "com.google.firebase:firebase-database:11.0.0"` in my `android/app/build.gradle`:
```
com.android.build.api.transform.TransformException: java.util.zip.ZipException: duplicate entry: com/google/android/gms/internal/aab.class
```
I tried to exclude various dependencies from other modules to no avail. Changing all of my Firebase and Play Services dependencies to `11.0.2` solved the issue.

2) iOS - When including the required React and Yoga pods for RNFirebase installation, Xcode gave me the following error when trying to build: `Linker command failed with exit code 1` with the issue `771 duplicate symbols for architecture x86_64`. By default, CocoaPods seemed to be installing React libraries for iOS 8.0 and these were the source of the extra symbols. I had no platform set in my Podfile and had to add the line `platform :ios, '9.0'` to the top of the file to resolve this.

3) iOS - I would suggest that the Manual Installation method described [here](https://github.com/invertase/react-native-firebase/blob/9f20e0bfe5f71821826d749ece27d3d99919ba96/docs/installation-ios.md) (2.3) be re-added to the docs. It worked well for me (with the extra step of explicitly adding `libRNFirebase.a` to Link Binary with Libraries) and allowed me to circumvent issue 2 above. The big advantage I see to this installation method is not having to deal with React and Yoga pods, and not needing to remember to keep them up to date whenever RN is upgraded.

I'd be happy to submit a PR for any or all of these fixes, if you guys would be interested.

### Environment

1. Target Platform (e.g. iOS, Android): iOS, Android
2. Development Operating System (e.g. macOS Sierra, Windows 10):, macOS Sierra
3. Build tools (Xcode or Android Studio version, iOS or Android SDK version, if relevant): Xcode 8.3.3, iOS 10.3, Android SDK 23
4. React Native version (e.g. 0.45.1): 0.42.3
5. RNFirebase Version (e.g. 2.0.2): 2.0.4

**Comment by harisvsulaiman** [comment]
Your app is using duplicate versions of the same dependency ( I had the same problem ).
Kindly refer to the FAQ page.
http://invertase.io/react-native-firebase/#/faqs

**Comment by chrisbianca** [comment]
@esamelson Thanks for raising this.  I have just ensured that 1 and 2 are both included in the upcoming v3 documentation.

As for 3, we're working on it - there are a few issues that have come about as a result of manually linking in the past, but I think we've resolved most of them for v3.  As soon as we're happy we'll add back in the instructions, as well as the option to use `react-native link`
